server/weaverbird/backends/sql_translator/README.md

```diff
@@ -0,0 +1,11 @@
+# SQL VQB Translator
```

Comment:
can you add a link or move this doc to the step documentation ? https://github.com/ToucanToco/weaverbird/blob/master/docs/_docs/tech/steps.md#convert-step

Reply:
:+1: